exercises/practice/parallel-letter-frequency/.docs/instructions.append.md

```diff
@@ -1,7 +1,11 @@
 # Instructions append
 
-Single-threaded (non-concurrent) solutions can pass all tests [but the last.](https://www.youtube.com/watch?v=mJZZNHekEQw)  Your solution will be tested for concurrency by submitting it as a [Runnable](https://docs.oracle.com/javase/7/docs/api/java/lang/Runnable.html) to an [ExecutorService.](https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/ExecutorService.html) Your solution must leverage multiple [Threads](https://docs.oracle.com/javase/7/docs/api/java/lang/Thread.html) to pass the final test.
+The automated tests are currently not checking for parallelism. Please ensure yourself that your solution is using some form of parallelism, like threads or parallel streams.
```

Comment:
I'm not sure if adding that the tests do not check for parallelism is a great move, maybe we should remove the 

> Your solution will be tested for concurrency by submitting it as a [Runnable](https://docs.oracle.com/javase/7/docs/api/java/lang/Runnable.html) to an [ExecutorService.](https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/ExecutorService.html) Your solution must leverage multiple [Threads](https://docs.oracle.com/javase/7/docs/api/java/lang/Thread.html) to pass the final test

And then when we create the analyzer for this exercise then try to check that they are using parellelism and if not leave a comment, @sanderploegsma  what are your thoughts on this one?

Reply:
Yes I agree, I don't think the instructions should explicitly mention that parallelism isn't tested. My 2cts:

1. Remove the first paragraph completely.
2. The list of resources look good, but I would move them to a new file `hints.md`, so that students can try to solve the exercise on their own (without spoilers, if you will) if they want to.

Reply:
Sounds good, I can take care of that.

Will a new file `hints.md` automatically be picked up and included, or does it have to be configured somewhere?

Reply:
I guess `hints.md` will get picked up, others exercises are using it too.

I've updated the PR to remove the paragraph that mentions that parallelism isn't tested and move the links over to `hints.md`.

Reply:
Yup, the `hints.md` file will automatically be picked up by the Exercism tooling, no need to configure it anywhere!

Here's a reference to the relevant documentation in case you're interested: https://exercism.org/docs/building/tracks/practice-exercises#h-documentation-files